packages/toc/package.json

```
@@ -52,12 +52,14 @@
     "@jupyterlab/rendermime": "^3.1.0-alpha.8",
     "@jupyterlab/translation": "^3.1.0-alpha.8",
     "@jupyterlab/ui-components": "^3.1.0-alpha.8",
+    "@lumino/commands": "^1.12.0",
     "@lumino/coreutils": "^1.5.3",
     "@lumino/messaging": "^1.4.3",
     "@lumino/signaling": "^1.4.3",
     "@lumino/widgets": "^1.19.0",
     "react": "^17.0.1",
-    "react-dom": "^17.0.1"
+    "react-dom": "^17.0.1",
+    "typescript": "~4.1.3"
```

No longer needed :tada: 
```suggestion
    "react-dom": "^17.0.1"
```